continuity.wessbas/continuity.wessbas.gradle

```diff
@@ -12,9 +12,11 @@ dependencies {
 	compile project(':continuity.api')
 	compile project(':continuity.commons')
 	
+	compile files('lib/behaviormodelextractor_forecast.jar')
```

Comment:
We should upload this to the Nexus as a new version

Reply:
+1 I also need this for my workload modularization approach